docs/audit_report/src/00_09_introduction.rst

```diff
@@ -50,162 +50,167 @@ unit and integration tests (in ``src/tests``, ``src/bogo_shim``,
 ``src/build-data``), and documentation (in ``src/doc``).
 
 The review in this document keeps track of changes in all the above-mentioned
-components. For the library implementation itself (``src/lib``), all modules that
-are *required* or *available* in the BSI build policy and their dependencies are
-in the scope of this document. Additionally, we review the following modules and
-its dependencies: `getentropy`, `ffi`, `xts`, `pkcs11`, `tls12`, `tls13`,
-`tls_cbc`, `x509`, `certstor_windows`, `certstor_macos`, `certstor_flatfile`,
-`certstor_sql`, `certstor_sqlite3`, `certstor_system_macos`, `certstor_system_windows`,
-`dilithium`, `dilithium_aes`, `dilithium_common`,
-`kyber`, `kyber_90s`, `kyber_common`,
-`sha1_armv8`, `sha1_sse2`, `sha1_x86`,
-`sphincsplus_common`, `sphincsplus_sha2`, `sphincsplus_shake`.
-Patches that don't alter any of the above-mentioned components or relevant
-modules are considered out-of-scope.
+components. For the library implementation itself (``src/lib``), all modules
+that are *required* or *available* in the BSI build policy and their
+dependencies are in the scope of this document. Additionally, we review the
+following modules and its dependencies: ``aes_armv8``, ``aes_ni``,
+``aes_power8``, ``aes_vperm``, ``argon2_avx2``, ``argon2_ssse3``,
+``certstor_flatfile``, ``certstor_sql``, ``certstor_sqlite3``,
+``certstor_system``, ``certstor_system_macos``, ``certstor_system_windows``,
+``dilithium``, ``dilithium_aes``, ``ffi``, ``ghash_cpu``, ``ghash_vperm``,
+``keccak_perm_bmi2``, ``kyber``, ``kyber_90s``, ``pkcs11``, ``sha1_armv8``,
+``sha1_sse2``, ``sha1_x86``, ``sha2_32_armv8``, ``sha2_32_bmi2``,
+``sha2_32_x86``, ``sha2_64_bmi2``, ``shake``, ``sphincsplus_sha2``,
+``sphincsplus_shake``, ``tls_cbc``, ``tls12``, ``tls13``, ``tls13_pqc``,
+``xts``. Patches that don't alter any of the above-mentioned components or
+relevant modules are considered out-of-scope.
```

Comment:
This list of additional modules also contains the `if_available` modules of the BSI policy, like `aes_armv8`, for example. We do not want that, do we?

Reply:
I think the following (platform dependent) modules should still be in this list, as they are not in the BSI policy nor a requirement of any of the modules:

- `certstor_system_macos`
- `certstor_system_windows`
- `certstor_sqlite3`
- `sha1_armv8`
- `sha1_sse2`
- `sha1_x86`

Basically all modules that we review but that would need to be explicitly mentioned in `--enable-modules=` to be enabled.

Reply:
Mhm, fair enough. I'll open a pull request for those.